### DISCUSSION TITLE
New duplicates from darkroom

### DISCUSSION THREAD

**Discussion by aurelienpierre** [aurelienpierreeng/ansel#804] [Announcements]
Should new duplicates created from darkroom be opened in darkroom straight away ? If not, the GUI is not clear regarding what duplicate is active (selected/hover color to revise for clarity).

Note for self: need to unfuck the double selection API first, there is a collision between "selected in darkroom" (active image) and "selected on user request" (click or space bar over a thumbnail).

**Comment by Jiyone** [comment]
> Should new duplicates created from darkroom be opened in darkroom straight away ?

I think it could be opened if we are in Darkroom, shouldn't if we are in Lighttable but it should be selected as well as the original thumbnail.

**Comment by pedrorrodriguez** [comment]
Agree with @Jiyone open if created from darkroom, don't if created from lighttable
